tests/cn/division_types.error.c

```diff
@@ -0,0 +1,25 @@
+/*  NOTE: 
+    Division should be done by ints of the same sign and size.           
+    This will all fail with type error since they are ints of different signs and sizes.
```

Comment:
Not quite, they'll fail because the return type is not right. As mentioned earlier, one can divide by numbers of different signs and types carefully.